src/encoded/submission_status.py

```diff
@@ -57,10 +58,31 @@ def get_submission_status(context, request):
         )
         search_res = search(context, subreq)["@graph"]
         file_sets = []
+        file_group_color_map = {}
         for res in search_res:
             file_set = res
             file_set["submitted_files"] = process_files_metadata(res.get("files", []))
+
+            if "file_group" in file_set:
+                fg = file_set["file_group"]
+                fg_str = f"{fg['submission_center']}_{fg['sample_source']}_{fg['sequencing']}_{fg['assay']}"
+                file_set["file_group"] = fg_str
+                # Place holder that will be replaced in the next step
+                file_group_color_map[fg_str] = None
+            else:
+                file_set["file_group"] = "No file group assigned"
+                file_set["file_group_color"] = "#eeeeee"
```

Comment:
I'd strongly prefer you create a top level "color map" that maps colors to their IDs here so it is more clear what colors you're using where.